test/library/packages/LinearAlgebra/correctness/testCholesky.chpl

```diff
@@ -0,0 +1,52 @@
+use LinearAlgebra;
+/*
+proc cholesky(A: [] real, uplo: string = 'L') where A.rank == 2 {
```

Comment:
Did you mean to remove this comment block?

Reply:
Yes, I did mean to remove it.  Gone now.